test/math.js

```diff
@@ -439,6 +439,14 @@ describe('Math', function () {
       expect(Object.is(-0, Math.round(-0.5 + Number.EPSILON / 3.99))).to.equal(true);
       expect(Object.is(-0, Math.round(0 - Number.EPSILON / 3.99))).to.equal(true);
     });
+
+    it('returns 1 / Number.EPSILON + 1 for 1 / Number.EPSILON + 1', function () {
+      expect(Math.round(1 / Number.EPSILON + 1)).to.equal(1 / Number.EPSILON + 1);
```

Comment:
Safari 8, IE 11, Opear 12 and FF < 12 (?) are buggy here, 

Spec says "Returns the Number value that is closest to x and is equal to a mathematical integer"
All values greater than `4503599627370496` are already integers. Browsers are buggy, because internally they do `Math.floor(x + 0.5)`, possibly, so the result is converted to the next integer sometimes.